Filters are now hidden on mobile if there are no results

jpggvilaca opened the pull request:
resolve #433 .

Filters are now hidden on mobile if there are no results and the dropdown is closed.

tureey commented on src/components/Filter/Filter.css:
```diff
@@ -39,6 +39,11 @@
     width: 100%;
   }
+
+  .inputs[aria-expanded="false"] {
```
It's ironic that we are using ARIA attributes for accessibility... but display: none; makes the content not accessible for screen readers.

Should we create a util class for "hiding"? So we stop using `display: none`

moshfeu replied:
I'm not sure I understand. When the inputs are hidden we don't want them to be accessible, isn't?

jpggvilaca commented:
@moshfeu I'll improve that later tonight then.

@tureey you have a point. I think css wise the app is a bit of a mess. What I can do is to use `opacity` for now, instead of `display. WDYT?

jpggvilaca commented:
@moshfeu can you pull and check again? Fixed animation overlap; changed `display` to `opacity`. Add a little padding to the inputs section to avoid them to be glued to the viewport boundaries.

tureey commented:
IMHO to do nor block the PR, I would merge this once what @moshfeu requested, and open a new PR to change what I proposed.

If using `opacity` it's solved and acts as expected, it's perfect!

moshfeu commented:
@tureey if you'll approve the PR I (tomottow) or @crysfel will merge it.
Thanks!
